core/src/main/scala/org/apache/spark/rpc/netty/NettyStreamManager.scala

```diff
@@ -19,6 +19,7 @@ package org.apache.spark.rpc.netty
 import java.io.File
 import java.util.concurrent.ConcurrentHashMap
 
+import org.apache.spark.SparkContext
```

Comment:
Why did we add this?

Reply:
We refer to `[[SparkContext.addFile]]` below in scala doc. This import fix the link, otherwise scala doc cannot find where `[[SparkContext.addFile]]` should be linked to.

As the other comment of yours, adding this doesn't affect production code logic though.

Reply:
Yeah, this is a big general problem with the scaladoc. It requires imports that aren't there. I don't mind importing this but it doesn't solve the overall issue.

Reply:
I personally work around this by fully qualified path in the doc though. I don't think this is something worth fixing alone.